octobot_commons/symbol_util.py

```diff
@@ -39,6 +39,16 @@ def merge_symbol(symbol):
     return symbol.replace(MARKET_SEPARATOR, "")
 
 
+def create_pair(base, quote):
```

Comment:
it's not already the job of the function just behind ?

Reply:
oh right ! I'm blind :p